<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QWidget" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1256</width>
    <height>779</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="gridBackgroundLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>591</width>
     <height>591</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="tableLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1271</width>
     <height>781</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>670</y>
     <width>141</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border: none;
    background: transparent;
}
QPushButton:pressed {
    background: transparent;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>800</width>
     <height>800</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="nextButton">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>670</y>
     <width>131</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border: none;
    background: transparent;
}
QPushButton:pressed {
    background: transparent;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>800</width>
     <height>800</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="largeShip">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>230</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="tinyShip">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>240</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="statusLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>620</y>
     <width>901</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 18pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Player one, set your board. Don't let that sneaky player two peek at it!</string>
   </property>
  </widget>
  <zorder>tableLabel</zorder>
  <zorder>gridBackgroundLabel</zorder>
  <zorder>backButton</zorder>
  <zorder>nextButton</zorder>
  <zorder>largeShip</zorder>
  <zorder>tinyShip</zorder>
  <zorder>statusLabel</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
